assets/oag.css

```diff
@@ -657,6 +664,18 @@
 		box-shadow: var(--oag-shadow);
 	}
 
+	.row.gutter-35  {
+		/* Add gutter spacing for rows */
+		margin-left: -17.5px;
+		margin-right: -17.5px;
+
+		& > [class*="col-"] {		
+			margin-bottom: 10px;
+			padding-left: 17.5px;
+			padding-right: 17.5px;	
+		}
+	}
+
```

Comment:
Remove this. Let's push back on this. I don't see the point of adding custom CSS for a mere 5px of gutter.